src/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -106,4 +106,4 @@ export const SearchBlockViewComponent = compose(
   (Component) => React.memo(Component, blockPropsAreChanged),
 )(SearchBlockView);
 
```

Comment:
Why did you remove the compose?

Reply:
https://github.com/plone/volto/pull/5340#discussion_r1375475109